website/src/components/Changelog/Apple.tsx

```diff
@@ -25,6 +25,10 @@ export default function Apple() {
     <Entries downloadLinks={downloadLinks} title="macOS / iOS">
       {/* When you cut a release, remove any solved issues from the "known issues" lists over in `client-apps`. This must not be done when the issue's PR merges. */}
       <Unreleased>
+        <ChangeItem pull="9308">
+          Fixes a minor crash in the network extension that could occur when
```

Comment:
I don't think there is such a thing as a minor crash. Maybe change that to rare?

Reply:
Yeah I can update it. The crash only reliably occurred during sign out, so no user impact.